actors/miner/src/lib.rs

```diff
@@ -2105,12 +2115,19 @@ impl Actor {
                     "failed to load pre-committed sectors",
                 )
             })?;
-        confirm_sector_proofs_valid_internal(
+
+        let sector_activations: Vec<SectorActivation> =
+            precommited_sectors.iter().map(|x| x.clone().into()).collect();
+        let rew = request_current_epoch_block_reward(rt)?;
+        let pwr = request_current_total_power(rt)?;
+        let info = get_miner_info(rt.store(), &st)?;
```

Comment:
These values were passed from the power actor to avoid each miner needing to make these calls. Don't regress that.